ModelImporter.cpp

```diff
@@ -85,12 +85,23 @@ Status importInput(ImporterContext* importer_ctx,
 
 Status importInputs(ImporterContext* importer_ctx,
                     ::ONNX_NAMESPACE::GraphProto const& graph,
-                    string_map<TensorOrWeights>* tensors) {
+                    string_map<TensorOrWeights>* tensors,
+                    uint32_t weights_count,
+                    onnxTensorDescriptor const* weight_descriptors) {
+  // The weights may come from two sources:
```

Comment:
Note: in ONNXIFI, weights may come from either initializer list, or weight descriptors, but not both. ONNXIFI implementation should return an error code if both are specified.

Reply:
Hmmm. Why do we have this constraint?

Reply:
To simplify implementation of ONNXIFI backend, and push ONNXIFI users to pass everything in weight descriptors.

Reply:
OK. I don't have a strong opinion on this actually. Let's change.